src/Sylius/Bundle/ProductBundle/Resources/config/validation/Product.xml

```diff
@@ -33,7 +33,6 @@
             </constraint>
         </property>
         <getter property="variants">
-            <constraint name="Valid" />
```

Comment:
I moved this constraint to [here](https://github.com/Sylius/Sylius/pull/7135/files#diff-64aa754ad20c13ec011448205027e452R54), as it's only needed for simple product (even more - it destroys the universe if it's validated on configurable one 🐃 )